car-service/src/main/kotlin/com/curious/programmers/carservice/configuration/security/SecurityConfiguration.kt

```diff
@@ -0,0 +1,74 @@
+package com.curious.programmers.carservice.configuration.security
+
+import io.jsonwebtoken.security.Keys
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.authentication.ReactiveAuthenticationManager
+import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity
+import org.springframework.security.config.web.server.ServerHttpSecurity
+import org.springframework.security.crypto.factory.PasswordEncoderFactories
+import org.springframework.security.crypto.password.PasswordEncoder
+import org.springframework.security.web.server.SecurityWebFilterChain
+
+
+@Configuration
+@EnableWebFluxSecurity
+class SecurityConfiguration {
+
+
+    @Value("\${jwt.secret}")
+    var secret: String? = null
+
+    @Value("\${jwt.expiration}")
+    var expiration: Long? = null
```

Comment:
Maybe we could move injection of this parameters to the proper factory method argument list?

And I think we could assume that without those value, application should not start - long story short, as for me at least, we should not take into account `null`s here.